pkgs/development/python-modules/pyregion/default.nix

```diff
@@ -25,6 +27,29 @@ buildPythonPackage rec {
     astropy
   ];
 
+  # Upstream patch needed for the test to pass
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/astropy/pyregion/pull/136.patch";
```

Comment:
It's nice to use the `name` attribute to give the patch a brief description. e.g. in this case maybe `conftest-astropy-3-fix.patch`.

Reply:
Thanks, done.